runtime/compiler/CMakeLists.txt

```diff
@@ -67,6 +67,42 @@ add_custom_command(
 )
 add_custom_target(j9jit_tracegen DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/env/ut_j9jit.h)
 
+# JITSERVER_SUPPORT and protobuf
+if(${JITSERVER_SUPPORT})
```

Comment:
I think all instances of `if(${JITSERVER_SUPPORT})` can be simplified to just `if(JITSERVER_SUPPORT)`.